src/components/RepeatableActionsField/RepeatableActionsField.test.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+
+import { noop } from 'lodash';
+
+import { renderWithIntl } from '@folio/stripes-data-transfer-components/test/jest/helpers';
+
+import '../../../test/jest/__mock__';
+
+import { fireEvent } from '@testing-library/react';
```

Comment:
Move this to `react` imports, please
See code style for imports sequence here - https://wiki.folio.org/pages/resumedraft.action?draftId=14458784&draftShareId=483fa2fb-615e-4002-a916-c976bd5fb8c6&